src/language-js/embed.js

```diff
@@ -20,6 +20,19 @@ function embed(path) {
     return;
   }
 
+  const formatter = getFormatter(path);
+
+  if (formatter) {
+    // Special case: whitespace-only template literals
+    if (node.quasis.length === 1 && node.quasis[0].value.raw.trim() === "") {
+      return "``";
+    }
+
+    return formatter;
+  }
+}
+
+function getFormatter(path) {
```

Comment:
Or would it be better to use the word "embedder" and rename the functions to `embedCss`, `embedHtml`, etc?